pegasus/sites.v3/code.org/views/donor_slider.haml

```diff
@@ -21,3 +20,9 @@
             = "#{supporter[:name_s]}"
           - else
             != "#{supporter[:name_s]}, "
+        :javascript
```

Comment:
Linking external resources pauses page-loading, and would negatively impact performance.